src/data/markdown/translated-guides/en/06 Testing Guides/04 Running large tests.md

```diff
@@ -5,18 +5,17 @@ excerpt: 'How to run large-scale k6 tests without distributed-execution'
 
 This document explains how to launch a large-scale k6 test on a single machine without the need for distributed execution.
```

Comment:
Maybe we should prepare readers for what they are going to read and what they need?

```
This document explains how to launch a large-scale k6 test on a single machine without the need for distributed execution. It most cases, the process involves the following:
- Shell commands to increase the default network and user limits of the operating system
- Monitoring to ensure safe resource usage from the load generator
- Efficient test design, with attention to scripting, k6 options, and file uploads.
- Some error monitoring to detect network and SUT failures.
```

Something like this would serve as a TL;DR, and it would prepare readers for what they need to do (as it's more than just adding a flag,  `--iterations 1000000`)

Reply:
Sure, see 8d725942. I changed some of the wording slightly.